dask_cuda/tests/test_dask_cuda_worker.py

```diff
@@ -194,18 +208,20 @@ def test_pre_import(loop):  # noqa: F811
     module = None
 
     # Pick a module that isn't currently loaded
-    for m in pkg_resources.working_set:
-        if m.key not in sys.modules.keys():
-            module = m.key
+    for m in pkgutil.iter_modules():
+        if m.ispkg and m.name not in sys.modules.keys():
+            module = m.name
```

Comment:
Changed this because IIUC `pkg_resources.working_set` is listing the installed distributions and not necessarily the importable modules; this becomes an issue if the distribution and module names aren't the same (i.e. one would `conda install pillow` and then `import PIL`), which was causing some failures in CI that seem unrelated to the changes here.

Reply:
Decided to break this off into https://github.com/rapidsai/dask-cuda/pull/1085 since it seems like this issue is blocking CI elsewhere (https://github.com/rapidsai/dask-cuda/pull/1081 for example)

Reply:
Thanks @charlesbluca , the fix should get merged as soon as tests pass.

Reply:
Had a question about using [`importlib.metadata`]( https://docs.python.org/3/library/importlib.metadata.html ) from the Python standard library ( https://github.com/rapidsai/dask-cuda/pull/1085#issuecomment-1385921939 )